### PR TITLE
Improve the Privacy and Security section.

### PR DESCRIPTION
**Pull request by darktears**
Closes #???

The following tasks have been completed:

 * [ ] Modified Web platform tests (link to pull request)

Implementation commitment:

 * [ ] Chromium (https://bugs.chromium.org/p/chromium/issues/detail?id=) 
 * Gecko - Waiting on [standards position](https://github.com/mozilla/standards-positions/issues/210) before committing.
 * WebKit - Not participating in this working group.

***
<a href="https://pr-preview.s3.amazonaws.com/darktears/screen-fold/pull/42.html" title="Last updated on Nov 18, 2020, 4:03 PM UTC (0639879)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/screen-fold/42/1910418...darktears:0639879.html" title="Last updated on Nov 18, 2020, 4:03 PM UTC (0639879)">Diff</a>

### CODE REVIEW

**Comment by darktears** [comment]
@anssiko @kenchris @diekus-zz PTAL

File: index.html [hunk 1]
```diff
@@ -637,30 +637,154 @@ <h2>
         Security and Privacy considerations
       </h2>
       <p>
-        Exposing the screen fold angle can have some privacy implications when it
-        comes to privacy and security. Specifically, the angle value could be used
-        for browser fingerprinting, in a way that can help fully or partially
-        identify individual users. It is up to the implementer to minimize this,
-        and some proposed solutions are listed below.
+        The Screen Fold API exposes two types of information:
```
Comment (kenchris) [inline index.html:640]: kinds of ?

File: index.html [hunk 2]
```diff
@@ -637,30 +637,154 @@ <h2>
...
+        <ol class="algorithm">
+          <li>
+            an |angle| value coming from the hinge position
```
Comment (kenchris) [inline index.html:643]: representing the

File: index.html [hunk 3]
```diff
@@ -637,30 +637,154 @@ <h2>
...
+          </li>
+          <li>
+            A |posture| determined from the angle value
```
Comment (kenchris) [inline index.html:646]: |angle| so that it links to the above?

**Comment by anssiko** [comment]
PTAL @diekus

File: index.html [hunk 4]
```diff
@@ -637,30 +637,155 @@ <h2>
...
-        Lowering the resolution of the angle MAY lead to reducing the uniqueness
-        that the value can have. Nonetheless, if several sites are being displayed
-        on the same device, it can be easy to identify that it is the same user,
-        even if the value is rounded. As a solution, applying some fuzziness to
-        the value itself MAY work.
+        Typical sensor readings are sent at a constant frequency to whomever is listening to its readings.
+        However the fold angle only communicates its value when the hinge is manipulated by the user.
+        Variations in the angle’s readings and posture calculation, as well as event dispatching frequency
+        offer a possibility of fingerprinting to identify users. User agents may reduce this risk by limiting
+        or coalishing events when exposing this information to web developers. Users don’t constantly adjust
```
Comment (anssiko) [inline index.html:656]: coalescing

**Review by anssiko — APPROVED** [review]

**Comment by darktears** [comment]
Thanks @anssiko. Feel free to merge whenever you'd like.

**Comment by darktears** [comment]
@lauramorinigo any feedback?

**Comment by anssiko** [comment]
@darktears I’ll let @kenchris merge when he’s happy. LGTM.

Any further feedback we can incorporate in updates. This is a good first stab.

File: index.html [hunk 5]
```diff
@@ -637,30 +637,155 @@ <h2>
...
-        Exposing the screen fold angle can have some privacy implications when it
-        comes to privacy and security. Specifically, the angle value could be used
-        for browser fingerprinting, in a way that can help fully or partially
-        identify individual users. It is up to the implementer to minimize this,
-        and some proposed solutions are listed below.
+        The Screen Fold API exposes two kinds of information:
+        <ol class="algorithm">
+          <li>
+            an <a href="#the-angle-attribute-get-current-screen-fold-angle">angle</a> value
+            representing the hinge position
```
Comment (kenchris) [inline index.html:644]: add dot at end

**Review by kenchris — APPROVED** [review]

**Comment by LaszloGombos** [comment]
This mitigation looks good to me. Thanks !

**Pull request merged**